tests/test_basic.py

```diff
@@ -5,11 +5,67 @@
 import unittest
 
 
+import crash_similarity
+
+
 class BasicTest(unittest.TestCase):
 
-    def test_pass(self):
+    # Train Model to be used in all tests
+    paths = ['crashsimilarity_data/test.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-09.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-08.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-07.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-06.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-05.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-04.json.gz', 'crashsimilarity_data/firefox-crashes-2016-11-03.json.gz']
```

Comment:
would not be insufficient data to train the model?
For now I only insert 6 entries.